packages/perseus/src/widgets/interactive-graphs/graphs/components/angle.tsx

```diff
@@ -82,7 +84,9 @@ export const Angle = ({
         angleInDegrees = 360 - angleInDegrees;
     }
 
-    const angleLabelNumber = parseFloat(angleInDegrees.toFixed(1));
+    const angleLabelNumber = parseFloat(
+        angleInDegrees.toFixed(snapTo === "angles" ? 0 : 1),
```

Comment:
Non-blocking nit: It might be good to add a comment here explaining that we only want to show whole angles in angle mode. :)